web-ng/src/app/components/project-header/project-header.component.ts

```diff
@@ -94,6 +97,15 @@ export class ProjectHeaderComponent implements OnInit, OnDestroy {
     });
   }
 
+  initPermission(): void {
+    const project = this.projectService.getCurrentProject();
+    if (!project) {
+      return;
+    }
+    const acl = this.projectService.getProjectAcl(project);
```

Comment:
Why not just have `getProjectAcl` return the ACL of the current project? It can return an empty ACL if no project is loaded.